packages/vite/src/node/plugins/css.ts

```diff
@@ -2517,7 +2526,21 @@ const scssProcessor = (
       worker?.stop()
     },
     async process(environment, source, root, options, resolvers) {
-      const sassPackage = loadSassPackage(root)
+      let failedSassEmbedded: boolean | undefined
+      let sassPackage = loadSassPackage(root, failedSassEmbedded ?? false)
+
+      if (failedSassEmbedded === undefined) {
+        // failedSassEmbedded is undefined for the first time
```

Comment:
```suggestion
      let sassPackage = loadSassPackage(root, failedSassEmbedded ?? false)
      // failedSassEmbedded is undefined for the first time
      if (failedSassEmbedded === undefined) {
        failedSassEmbedded = false
```
I think we need to set `failedSassEmbedded` here and hoist `let failedSassEmbedded: boolean | undefined` above `return {` so that we don't run this `if (failedSassEmbedded === undefined)` block more than once.